packages/rancher-istio-1.8/charts/values.yaml

```diff
@@ -82,7 +82,11 @@ kiali:
       custom_metrics_url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
       url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
     tracing:
-      in_cluster_url: "http://tracing.istio-system.svc:16686/jaeger"
+      auth:
+        insecure_skip_verify: true
+      # this configuration needs changed to match your current installation for it to work
+      in_cluster_url: "https://tracing-service:8080/jaeger"
+      url: "https://tracing-service:8080/jaeger"
```

Comment:
Is this off by default ? Or is that actually a functioning url.

Reply:
Tracing is disabled by default. With that in mind, I have updated this to both be empty strings since we cannot set them to a default value and they don't need set until tracing is enabled.